Add more Character Backstories

kLinQin opened the pull request:
#3

kLinQin committed:
Create Yinlin.md

kLinQin committed:
Update Yinlin.md

kLinQin committed:
Update Yinlin backstory (done).md

kLinQin committed:
Create Yuanwu.md

kLinQin committed:
Merge branch 'Iqrar99:main' into main

kLinQin committed:
Merge branch 'Iqrar99:main' into main

kLinQin committed:
update Backstory All character

kLinQin committed:
Merge branch 'main' of https://github.com/kLinQin/wwi

kLinQin committed:
FIx Typo Chalcharo and Changli Description

kLinQin committed:
Update calcharo.md

Iqrar99 commented on docs/ww/lore/character-backstory/Camellya.md:
```diff
@@ -0,0 +1,331 @@
+
+gumamnya. Dia merasa stabil, baik secara pikiran maupun tubuh, dan pikiran tentang membutuhkan pod medis terasa begitu jauh. Ini adalah hal yang baik.
+
+Dia tahu **{PlayerName}** akan tetap berada di **Black Shores**, mempersiapkan perjalanan **{Male=his;Female=her}** ke **Rinascita**. Prospek itu menyenangkannya, karena itu berarti lebih banyak waktu untuk berbicara, tertawa, dan menghabiskan waktu bersama.
```
Sebaiknya setiap mention {Playername} diganti dengan `Rover`

Iqrar99 commented on docs/ww/lore/character-backstory/Camellya.md:
```diff
@@ -0,0 +1,331 @@
+gumamnya.\
+Memikirkan keselamatan seseorang, atau apakah mereka bahkan ada... benar-benar perasaan yang baru! Pasti itu berarti dia peduli dengan kehidupan mereka. Pemikiran seperti itu terasa asing—dia tidak pernah peduli dengan hidupnya sendiri, atau hidup orang lain. Namun, setelah meninggalkan **Stellar Matrix** bersama **{PlayerName}** dan menemukan identitasnya sebagai **Lady Flora**, dia mulai meliha ...
+
+Namun, dia tidak berniat mengikuti **{PlayerName}** ke **Rinascita**. Pandangannya tentang beberapa aspek sejarah **Solaris** berbeda, dan sama seperti dia tidak berniat mengganggu pilihan **{Male=his;Female=her}**, dia juga tidak merasa perlu untuk menjelaskan pilihannya sendiri. Bahkan, untuk menghindari beban tambahan bagi **{PlayerName}**, dia menyembunyikan beberapa kebenaran dari **{Male=hi ...
```
untuk semua parameter `{Male=him;Female=her}` disesuaikan ke bahasa indonesia seperti penggunaan imbuhan `-nya` atau `dia`

Iqrar99 reviewed (changes requested):
Silakan cek dulu semua file yang memiliki parameter tersebut

kLinQin committed:
Update Camellya.md

1. change: {player name} to {Rover}
2. change: {she ; he} to {-nya}

kLinQin committed:
Update Camellya.md

edit Cherished Items

kLinQin committed:
Fixing Typo

kLinQin committed:
Fix Typo
change text format
change {player name} to {rover}
change {male : Female} to {-nya / dia}

kLinQin committed:
Add Xiangli yao backstory
